### DISCUSSION TITLE
Which Minecraft version is most remapped?

### DISCUSSION THREAD

**Discussion by kangarko** [MCPHackers/RetroMCP-Java#87] [General]
I noticed the mappings vary across versions. Do you know which Minecraft version or versions have the most remapped stuff? Thanks.

**Comment by PhoenixVX** [comment]
a1.1.2_01 is one of, if not the most mapped one currently. The unreleased mappings for Infdev and Classic are also 100% mapped for classes, methods, fields, and parameters.

Also thanks for using the discussion forums.

**Comment by kangarko** [comment]
You are welcome and thank you!
